test/utils_shared/test_utils.py

```diff
@@ -40,45 +40,91 @@ def get_example_job_as_dict(
     wsid: int = 123,
     authstrat: str = "kbaseworkspace",
     scheduler_id: str = None,
+    params: dict = None,
+    narrative_cell_info: dict = None,
+    source_ws_objects: list = None,
+    method_name: str = None,
+    app_id: str = None,
 ):
     job = (
         get_example_job(
-            user=user, wsid=wsid, authstrat=authstrat, scheduler_id=scheduler_id
+            user=user,
+            wsid=wsid,
+            authstrat=authstrat,
+            scheduler_id=scheduler_id,
+            params=params,
+            narrative_cell_info=narrative_cell_info,
+            source_ws_objects=source_ws_objects,
+            method_name=method_name,
+            app_id=app_id,
         )
         .to_mongo()
         .to_dict()
     )
+    # Copy fields to match run_job signature
+    job_input = job["job_input"]
+    job["meta"] = job_input["narrative_cell_info"]
+    job["narrative_cell_info"] = job_input["narrative_cell_info"]
+    job["params"] = job_input["params"]
+    job["source_ws_objects"] = job_input["source_ws_objects"]
     job["method"] = job["job_input"]["method"]
     job["app_id"] = job["job_input"]["app_id"]
     job["service_ver"] = job["job_input"]["service_ver"]
     return job
 
 
+def get_example_job_input(wsid, params=None, method_name=None, app_id=None):
+    if params == None:
+        params = {}
+
+    job_input = JobInput()
+    job_input.wsid = wsid
+
+    job_input.method = method_name or "module.method"
+    job_input.requested_release = "requested_release"
```

Comment:
Wasn't this field removed?

Reply:
Yes, I have merged develop into this branch now so it's removed.

Reply:
But it's still there...?

Reply:
Ah, I was looking at the model. Yea ok deleted it

Reply:
:+1: